docs/source/index.rst

```diff
@@ -0,0 +1,55 @@
+DeepSpeed-MII
+=============
+
+.. image:: ../images/mii-white.svg
+   :width: 600
+
+.. note::
+
+   This project is under active development.
+
+
+Introducing MII, an open-source Python library designed by DeepSpeed to
+democratize powerful model inference with a focus on high-throughput, low
+latency, and cost-effectiveness.
+
+MII v0.1 introduced several features as part of our `DeepSpeed-FastGen release
+<https://github.com/microsoft/DeepSpeed/tree/master/blogs/deepspeed-fastgen>`_
+such as blocked KV-caching, continuous batching, Dynamic SplitFuse, tensor
+parallelism, and high-performance CUDA kernels to support fast high throughput
+text-generation with LLMs. The latest version of MII delivers up to 2.5 times
+higher effective throughput compared to leading systems such as vLLM. For
+detailed performance results please see our `DeepSpeed-FastGen release blog
```

Comment:
FYI this is the same hyperlink as above in the same paragraph, not sure if one is meant to point to a different link?